examples/pipelines/pandas_pipeline/components/filter_images/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.8-slim
```

Comment:
Should we add `--platform=linux/arm64` here? To make sure those images can also be built from a mac

Reply:
can we add add this in the bash script of `build_images.sh` as a flag during build? I saw you had this in one your PRs. Then we don't have to remember about including it in every base image.